Hunting/Assets/Scripts/GameManager/NonVRGameManager.cs

```diff
@@ -1,15 +1,52 @@
 ﻿using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
 
 public class NonVRGameManager : MonoBehaviour
 {
-    # region MonoBehaviour Methods
+    #region Serialize Fields
+    [SerializeField]
+    private GameObject reticleGameObject = null;
+
+    [SerializeField]
+    private GameObject healthDisplayGameObject = null;
+
+    [SerializeField]
+    private GameObject scoreGameObject = null;
+
+    [SerializeField]
+    private GameObject endGameImageGameObject = null;
+
+    [SerializeField]
+    private Text endGameScoreText = null;
+    # endregion
 
+    #region MonoBehaviour Methods
     void Start()
     {
         Cursor.lockState = CursorLockMode.Locked;
 
         if (Debug.isDebugBuild)
+        {
             Debug.Log("cursor lock state is " + Cursor.lockState.ToString());
+
+            if (reticleGameObject == null)

```

Comment:
Do they not throw null references anyways if they are missing?

Reply:
Oh yes, they do. 

I just find it helpful to add these debug statements that show up at the start of the game if something goes wrong, instead of waiting till a wrong behaviour occur half way through the game, then try to fix errors using error messages there.

These are not really necessary, and can be removed if you want; they're just a personal preference, for convenience.